test/bpf/verifier-test.sh

```diff
@@ -6,11 +6,9 @@ set -eo pipefail
 
 DEV="cilium-probe"
 DIR=$(dirname $0)/../../bpf
-MAPTOOL=$(dirname $0)/../../tools/maptool/maptool
```

Comment:
We also need to remove `maptool` from `SUBDIRS` in `tools/Makefile`. There's at least the Travis build currently failing on this.